src/resolvers/projectResolver.test.ts

```diff
@@ -5505,6 +5510,133 @@ function projectBySlugTestCases() {
 
     await campaign.remove();
   });
+  it('should return projects including active SortField campaigns', async () => {
+    const projectWithCampaign = await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      slug: String(new Date().getTime()),
+    });
+    const campaign = await Campaign.create({
+      isActive: true,
+      type: CampaignType.SortField,
+      sortingField: CampaignSortingField.Newest,
+      slug: generateRandomString(),
+      title: 'title1',
+      description: 'description1',
+      photo: 'https://google.com',
+      order: 1,
+    }).save();
+    const result = await axios.post(graphqlUrl, {
+      query: fetchProjectsBySlugQuery,
```

Comment:
can you rename the fetchProjectsbySlugQuery as fetchProjectBySlugQuery without the s. 

That was my bad but its a bit confusing, I thought it returned multiple projects but its just 1.

Reply:
haahaa you are right, done.